### PR TITLE
Fix: staking reward calculation

### PR DESCRIPTION
**Pull request by bdeneux**
I think there is a little mistake in the staking reward calculation.

### CODE REVIEW

File: docs/whitepaper/tokenomics.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Block reward for Validators and Delegators will depend on the inflation number t
 - Year 3: 9.6%
 - Year 4: 7.68%
 - Year 5: 6.144%
-- Year X: X-1 * 0.8
+- Year X: 15 * 0.8^(X-1)
```
Comment (ccamel) [inline docs/whitepaper/tokenomics.md:91]: Yes. Good catch. The expression is quite vague, but I guess it is defined by recurrence and should be:

- Year X: Y<sub>X-1</sub> * 0.8

Comment (ccamel) [reply docs/whitepaper/tokenomics.md:91]: Your formula is also correct (if you specify the percentage correctly): 

- Year X: 15% * 0.8^(X-1)

**Comment by ccamel** [comment]
@bdeneux Based on your work, I have slightly restyled the table of inflations to show the values better, and also used [katex](https://katex.org/) (provided as a plugin for docusaurus)  to express the general calculation formula. Let me know if it okay for you.

image

**Comment by bdeneux** [comment]
@ccamel Nice, it's ok for me ! 👍

**Review by ccamel — APPROVED** [review]

**Review by amimart — APPROVED** [review]

**Pull request merged**